compute_tools/src/http/api.rs

```diff
@@ -48,7 +48,7 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
     match (req.method(), req.uri().path()) {
         // Serialized compute state.
         (&Method::GET, "/status") => {
-            info!("serving /status GET request");
+            debug!("serving /status GET request");
```

Comment:
Perhaps it would make more sense to move this out of the match arm and just `debug!` every URI path + method that we receive, but overall the change is good.